css-backgrounds-4/Overview.bs

```diff
@@ -242,8 +242,70 @@ Painting Area: the 'background-clip' property</h3>
 <h2 id="corners">
 Corners</h2>
 
-<h3 id=corner-sizing>
-Corner Sizing: the 'border-radius' property</h3>
+<h3 id="corner-sizing">
+Corner Sizing: the 'border-radius' and 'border-*-radius' shorthand properties</h3>
+
+<h4 id="physical-corner-sizing">
+Physical Corner Sizing:
+The 'border-top-radius', 'border-right-radius',
+'border-bottom-radius', 'border-left-radius' side shorthands</h4>
+
+	<pre class=propdef>
+		Name: border-top-radius, border-right-radius, border-bottom-radius, border-left-radius
+		Value: <<length-percentage [0,&infin;]>>{1,2} [ / <<length-percentage [0,&infin;]>>{1,2} ]?
+		Initial: 0
+		Applies to: all elements (but see prose)
+		Inherited: no
+		Percentages: Refer to corresponding dimension of the <a>border box</a>.
+		Computed value: see individual properties
+		Animatable: see individual properties
+	</pre>
+
+	<p>The 'border-*-radius' shorthands set the two 'border-*-*-radius'
+	longhand properties of the related side. If values are given before
+	and after the slash, then the values before the slash set the
+	horizontal radius and the values after the slash set the vertical radius.
+	If there is no slash, then the values set both radii equally.
+	The two values for the radii are given in the order
+	top-left, top-right for 'border-top-radius',
+	top-right, bottom-right for 'border-right-radius'
+	bottom-left, bottom-right for 'border-bottom-radius',
+	and top-left, bottom-left for 'border-left-radius'.
+	If the second value is omitted it is copied from the first.
+
+<h4 id="flow-relative-corner-sizing">
+Flow-Relative Corner Sizing:
+The 'border-block-start-radius', 'border-block-end-radius',
+'border-inline-start-radius', and 'border-inline-end-radius' side shorthands</h4>
+
+	<pre class=propdef>
+		Name: border-block-start-radius, border-block-end-radius, border-inline-start-radius, border-inline-end-radius
+		Value: <<length-percentage [0,&infin;]>>{1,2} [ / <<length-percentage [0,&infin;]>>{1,2} ]?
+		Initial: 0
+		Applies to: all elements (but see prose)
+		Inherited: no
+		Percentages: Refer to corresponding dimension of the <a>border box</a>.
+		Computed value: see individual properties
+		Animatable: see individual properties
+	</pre>
+
+	<p>These properties correspond to the 'border-top-radius',
+	'border-right-radius', 'border-bottom-radius', and 'border-left-radius'
+	shorthand properties.
```

Comment:
This implies that with `writing-mode: horizontal-tb; direction: rtl`,
```css
border-block-start-radius: 1px 2px;
```
should become
```css
border-top-radius: 1px 2px;
```
i.e.
```css
border-top-left-radius: 1px;
border-top-right-radius: 2px;
```
Instead, I think that
```css
border-block-start-radius: 1px 2px;
```
should expand to
```css
border-start-start-radius: 1px;
border-start-end-radius: 2px;
```
which then will map to
```css
border-top-right-radius: 1px;
border-top-left-radius: 2px;
```

This is because shorthands must be expanded into longhands at parse time, while logical-to-physical mapping happens at computed-value time. So we can't have logical shorthands mapping into physical shorthands. Shorthands just expand, it's the longhands that can have logical/physical pairings.